eoscostarica/lifebank
# feat(backend): add full donate route and send congratulation for bloo…

Send congratulation email

Resolves #700 

### What does this PR do?
Send email congratulation when completing a donation

### How should this be tested?
- Make run
- Sign in as Lifebank
- Send tokens to donor
- Check email
